src/postPrComment.js

```diff
@@ -31,10 +31,12 @@ export default async ({
           score: result.scores[current]
         });
       });
-
       // the url
       markdown += `\n\n${result.url}`;
 
+      // the emulatedformfactor
+      markdown += `\n\n Mode: **${result.emulatedFormFactor}**`;
```

Comment:
I don't think this will work because `emulatedFormFactor` won't be part of `result` (unless I'm mistaken). We should have a test for comments, so I opened [issue #33](https://github.com/foo-software/lighthouse-check/issues/33).

Reply:
i have added it in the localLighthouse function `return` object  and tested it, looks fine.
<img width="1026" alt="Screen Shot 2020-06-24 at 6 14 27 PM" src="https://user-images.githubusercontent.com/38260994/85573241-c5293700-b646-11ea-8241-76677878279d.png">

Reply:
Super minor - can we put the link below "Mode: mobile" @Mirzamohammad22?

Reply:
Sure